mm2src/mm2_main/src/lp_swap/maker_swap.rs

```diff
@@ -1875,6 +1875,21 @@ impl MakerSavedSwap {
             self.taker_coin_usd_price = Some(rates.rel);
         }
     }
+
+    pub fn swap_pubkeys(&self) -> SwapPubkeys {
+        let mut swap_pubkeys = SwapPubkeys::default();
+
+        // TODO: Adjust for private coins when/if they are braodcasted
+        for data in &self.events {
+            if let MakerSwapEvent::Started(started) = &data.event {
+                swap_pubkeys.maker = Some(started.my_persistent_pub);
+                swap_pubkeys.taker = started.maker_coin_htlc_pubkey;
```

Comment:
Aren't those 2 pubkeys the maker's? can you please add a unit test to test that the right pubkeys are saved to DB? you should use 3 nodes for the test (taker/maker/seednode) and use `stats_swap_status` RPC to check that the right pubkeys are saved to the seednode's DB.

Reply:
ok